frontend/src/pages/checklist/[filters].tsx

```diff
@@ -99,6 +99,33 @@ const ChecklistResults: FC<ChecklistResultsProps> = ({
     window.print()
   }
 
+  // handle expanding all task details on print and closing tasks that weren't previously open after print
+  useEffect(() => {
+    const beforePrint = (event: Event) => {
+      let details = [...document.querySelectorAll<HTMLDetailsElement>('details:not([open])')]
+      for (let e of details) {
+        e.setAttribute('open', '')
+        e.dataset.wasclosed = ''
+      }
+    }
+
+    const afterPrint = (event: Event) => {
+      let details = [...document.querySelectorAll<HTMLDetailsElement>('details[data-wasclosed]')]
+      for (let e of details) {
+        e.removeAttribute('open')
+        delete e.dataset.wasclosed
+      }
```

Comment:
Use `const` and close the `details` element by using `open` property setter with `false`.

```suggestion
      const details = document.querySelectorAll<HTMLDetailsElement>('details[data-wasclosed]')
      details.forEach((el) => {
        el.open = false
        delete el.dataset.wasclosed
      })
```